mail_show_follower/models/mail_mail.py

```diff
@@ -1,16 +1,62 @@
-from odoo import models
+from odoo import api, models
 
 
 class MailMail(models.Model):
     _inherit = "mail.mail"
 
-    def _send(self, auto_commit=False, raise_exception=False, smtp_session=None):
-        plain_text = (
-            '<div summary="o_mail_notification" style="padding: 0px; '
-            'font-size: 10px;"><b>CC</b>: %s<hr style="background-color:'
-            "rgb(204,204,204);border:medium none;clear:both;display:block;"
-            'font-size:0px;min-height:1px;line-height:0; margin:4px 0 12px 0;"></div>'
+    @api.model
+    def _build_cc_text(self, partners):
+        if not partners:
+            return ""
+
+        def get_ctx_param(ctx_key, default_parm):
+            if ctx_key in self.env.context:
+                return self.env.context[ctx_key]
+            return default_parm
+
+        def remove_p(markup_txt):
+            if markup_txt.startswith("<p>") and markup_txt.endswith("</p>"):
+                return markup_txt[3:-4]
+            return markup_txt
+
+        company = self.env.company
+        partner_format = get_ctx_param(
+            "partner_format", company.show_followers_partner_format
+        )
```

Comment:
As explained above, wrap it please:

```suggestion
        partner_format = markupsafe.escape(get_ctx_param(
            "partner_format", company.show_followers_partner_format
        ))
```